fedimintd/src/bin/main.rs

```diff
@@ -120,19 +120,17 @@ async fn main() -> anyhow::Result<()> {
 
     task_group.install_kill_handler();
 
-    let module_inits = ModuleInitRegistry::from([
-        (
-            "wallet",
-            Arc::new(WalletConfigGenerator) as Arc<dyn ModuleInit + Send + Sync>,
-        ),
-        ("mint", Arc::new(MintConfigGenerator)),
-        ("ln", Arc::new(LightningModuleConfigGen)),
+    let module_inits = ModuleInitRegistry::from(vec![
+        Arc::new(WalletConfigGenerator) as Arc<dyn ModuleInit + Send + Sync>,
+        Arc::new(MintConfigGenerator),
+        Arc::new(LightningModuleConfigGen),
```

Comment:
note to self: Eventually we might want some builders to avoid manual arc creation